web-lib-funcs.pl

```diff
@@ -10091,7 +10120,7 @@ =head2 guess_mime_type(filename, [default])
 sub guess_mime_type
 {
 my ($file, $def) = @_;
-if ($file =~ /\.([A-Za-z0-9\-]+)$/) {
+if ($file =~ /\.([A-Za-z0-9\-]+)(?:[?].*|)$/) {
```

Comment:
What case does this handle?

Reply:
> What case does this handle?

This covers a case with an image having a tracker, i.e. `https://example.com/image1.png?uid=1234` or something like that.

Reply:
I think this should be stripped off by the caller to guess_mime_type, not inside the function.

Reply:
It seems wrong to me for a function that's supposed to guess the MIME type of a filename to have special handling for URLs.

Reply:
No problem, I've reverted it.